components/presenters/ContentPanel/partials/StyledLeftCol.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components'
+import { selectors } from '@royalnavy/design-tokens'
+
+const { color, spacing, mq } = selectors
+
+export const StyledLeftCol = styled.div`
+  border-top: 4px solid ${color('neutral', '000')};
+  background-color: ${color('neutral', 'white')};
+  padding: ${spacing('12')} ${spacing('10')};
+  width: 100%;
+
+  ${mq({ gte: 's' })`
+    padding: ${spacing('14')};
+  `}
+
+  ${mq({ gte: 'xl' })`
+    width: 50%;
+  `}
+
+  @media only screen and (min-width: 1900px) {
```

Comment:
I am wondering if something can be done so this is more self documenting. Maybe adding the `1900` as a constant?